packages/network/warpsync/validation.go

```diff
@@ -0,0 +1,14 @@
+package warpsync
+
+import (
+	"github.com/iotaledger/hive.go/core/identity"
+
+	"github.com/iotaledger/goshimmer/packages/core/epoch"
+)
+
+func (p *Protocol) RequestEpochCommittment(ei epoch.Index, to ...identity.ID) {
```

Comment:
This file should not be modified, so it's most likely a result of an incorrect merge?